File: Data Structures & Algorithms/is-subsequence/submission-0.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        index = 0
        t_index = 0
        
        while index < len(s):
            if t_index > len(t) - 1:
                return False
            if s[index] == t[t_index]:
                index += 1
            
            t_index += 1
        return True
            

File: Data Structures & Algorithms/number-of-senior-citizens/submission-0.swift
class Solution {
    func countSeniors(_ details: [String]) -> Int {
        var count = 0
        for item in details {
            let start = item.index(item.startIndex, offsetBy: 11)
            let end = item.index(item.startIndex, offsetBy: 13)

            guard let age = Int(item[start..<end]) else {
                print("invalid number")
                return -1
            }

            if age > 60 {
                count += 1
            }
        }
        return count
    }
}
